### ISSUE TITLE
Merge scripts/ and bin/?

### ISSUE THREAD

**Issue by SaschaMann** [exercism/org-wide-files#231]
I just noticed we have two directories that contain scripts that are called from the workflows. [`scripts/`](https://github.com/exercism/org-wide-files/tree/main/scripts) and [`.github/bin/`](https://github.com/exercism/org-wide-files/tree/main/.github/bin).

Should we merge these into one?

**Comment by ErikSchierboom** [comment]
Yes please. If they're only called from the workflows, I prefer .github/bin

**Comment by SaschaMann** [comment]
I prefer scripts in this repo. The entire repo is about these workflows and scripts, I don't think they should be hidden when we can avoid it.

**Comment by ErikSchierboom** [comment]
🤷 I also don't really care.

**Comment by dsdeboer** [comment]
Is this something I can assist on? I see the comments and issue are a little old, but it seems like a small task

**Comment by ErikSchierboom** [comment]
@dsdeboer Sure. The `scripts/apply-track-config.jl` could be moved to the `.github/bin` directory. You'll also need to update the workflows that reference that script.
